server/repository/src/db_diesel/changelog/changelog.rs

```diff
@@ -0,0 +1,199 @@
+use crate::{
+    diesel_macros::apply_equal_filter, DBType, EqualFilter, RepositoryError, StorageConnection,
+};
+use diesel::{helper_types::IntoBoxed, prelude::*};
```

Comment:
This is relocated form changelog_row.rs, comments below will outline what's actually changed